.github/actions/godot-build/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Build Godot'
+description: 'Build Godot with the provided options'
+inputs:
+  target:
+    description: 'The scons target (debug/release_debug/release)'
+    required: true
+    default: 'debug'
+  tools:
+    description: 'If tools are to be built'
+    required: true
+    default: false
+  tests:
+    description: 'If tests are to be built'
+    required: true
+    default: false
```

Comment:
Could maybe be made optional, it's only used in a couple builds.

BTW, does it make sense to have `required: true` and a `default`?

Reply:
> BTW, does it make sense to have `required: true` and a `default`?

No, in fact, specifying tests is not required, and defaults to false.
I've removed the unneeded `default`